tailormap-components/projects/core/src/lib/shared/shared.module.ts

```diff
@@ -93,36 +93,37 @@ import {
     MatDatepickerModule,
   ],
   exports: [
-    ReactiveFormsModule,
-    FormsModule,
-    MatButtonModule,
-    MatCheckboxModule,
-    MatTreeModule,
-    MatIconModule,
-    MatFormFieldModule,
-    MatTabsModule,
-    MatInputModule,
-    MatTooltipModule,
-    MatSortModule,
-    MatSelectModule,
-    MatGridListModule,
-    MatSliderModule,
-    MatDividerModule,
-    MatButtonToggleModule,
-    MatListModule,
-    HttpClientModule,
-    MatStepperModule,
-    MatProgressSpinnerModule,
-    MatMenuModule,
-    MatDialogModule,
-    DialogCloseButtonComponent,
-    TreeComponent,
-    MatRadioModule,
-    MatProgressBarModule,
-    MatAutocompleteModule,
-    DragDropModule,
-    MatDatepickerModule,
-  ],
+        ReactiveFormsModule,
```

Comment:
oh no... why 4 spaces here? the rest is 2

Reply:
Dunno why, reverted it